README.md

```diff
@@ -92,16 +112,23 @@ At runtime, the value of `My.Assets.star` will be something like
   module.exports = {
     ...
     elmAssetsLoader: {
-      localPath: function(path) {
-        return path.replace(/^\/public\//, "")
+      localPath: function(url) {
+        // transform `url` to a local path that resolves to a file
+        return url.replace(/^\/public\//, "")
```

Comment:
When you tag a path (`AssetPath "/public/images/star.png"`), the tagged string is passed as-is to `localPath`: so whether or not it comes with a leading slash or not depends solely on what kind of paths exist in the project's code base.

I'm not sure if `/^\/?public\//` is a generally copy-pasteable regex. The purpose of using a relative path like `public/` is presumably to reference an asset relative to the current path: how to translate it to a resolvable path will depend on each project's path setup.

As a usage example, I'm going to err on the side of limiting the expected input to what's in the example.